packages/vue/src/components/organisms/SfGroupedProduct/SfGroupedProduct.stories.js

```diff
@@ -432,7 +443,7 @@ export const UseTitleSlot = (args, { argTypes }) => ({
   >
     <SfGroupedProductItem
       :qty="productQty"
-      @input="productQty = $event"
+      @input="inputHandler"
```

Comment:
question: in file 56 f.e you are also passing the $event to inputHandler - both are ok, but maybe we should keep consistency?